packages/images/lib/compare.js

```diff
@@ -20,8 +20,21 @@ const DEFAULT_MATCH_THRESHOLD = 0.4;
  * @param {string} secondImage - Base64-encoded image file.
  * All image formats, that OpenCV library itself accepts, are supported.
  * @param {?Object} options [{}] - The content of this dictionary depends
- * on the actual `mode` value. See the documentation on `appium-support`
- * module for more details.
+ * on the actual `mode` value.
+ * For MATCH_TEMPLATE_MODE:
```

Comment:
I still would prefer to not change that, since it is needed to update this list every time anything new is added to appium-support.